pet-api/pet-api/Domain/Entities/Product.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace pet_api.Domain.Entities
+{
+   public class Product
+   {
+        public int ProductId { get; set; }
```

Comment:
Rename it please to 'Id', it is related to the Product class and it is clear that it's a Product identifier.